smoke_tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+package-lock.json
```

Comment:
I'm wondering if I should have left this in 🤔

Reply:
Yeah, I never know whether to commit the lock file or not 🤷‍♂️ 
We do have the main project one under version control, so maybe this one should be as well? 🤷‍♂️